pacemaker/pacemaker.go

```diff
@@ -153,13 +160,17 @@ func (p *RoundRobin) startNewViewTimeout(stopContext context.Context) {
 	for {
 		select {
 		case <-p.resetTimer:
+			p.timeoutContextCancle()
```

Comment:
For the leader, context should not be cancelled here, as the timer is reset on `OnReceiveProposal`. (Line 130)